jwql/utils/maths.py

```diff
@@ -0,0 +1,154 @@
+"""Various math-related functions used by the ``jwql`` instrument monitors.
```

Comment:
@bhilbert4 I wonder if `calculations.py` would be a better name for this module.  What do you think? 

To me, something like 

```python
from jwql.utils.calculations import mean_image
```

reads a bit better than

```python
from jwql.utils.maths import mean_image
```

Reply:
Yeah, that sounds better. I'll switch it.